qiskit/providers/aer/noise/errors/standard_errors.py

```diff
@@ -144,129 +168,44 @@ def pauli_error(noise_ops, standard_gates=True):
-                elif pstr != 'I':
-                    raise NoiseError("Invalid Pauli string.")
-            if mat.size == 1:
-                prob_identity += prob
-            else:
-                circ = make_unitary_instruction(
-                    mat, qubits, standard_gates=False)
-                pauli_circs.append(circ)
-                pauli_probs.append(prob)
-    if prob_identity > 0:
-        pauli_probs.append(prob_identity)
-        pauli_circs.append([{"name": "id", "qubits": [0]}])
-
-    error = QuantumError(
-        zip(pauli_circs, pauli_probs), number_of_qubits=num_qubits)
-    return error
-
-
-def _pauli_error_standard(noise_ops, num_qubits):
-    """Return Pauli error as standard Pauli gate qobj instructions."""
-
-    def single_pauli(pauli):
-        if pauli == 'I':
-            return {'name': 'id'}
-        if pauli == 'X':
-            return {'name': 'x'}
-        if pauli == 'Y':
-            return {'name': 'y'}
-        if pauli == 'Z':
-            return {'name': 'z'}
-        raise NoiseError("Invalid Pauli string.")
-
-    prob_identity = 0.0
-    pauli_circuits = []
-    pauli_probs = []
-    for pauli, prob in noise_ops:
-        if prob > 0:
-            # Pauli strings go from qubit-0 on left to qubit-N on right
-            # but pauli ops are tensor product of qubit-N on left to qubit-0 on right
-            # We also drop identity operators to reduce dimension of matrix multiplication
-            circuit = []
-            if isinstance(pauli, Pauli):
-                pauli_str = pauli.to_label()
-            else:
-                pauli_str = pauli
-            for qubit, pstr in enumerate(reversed(pauli_str)):
-                if pstr in ['X', 'Y', 'Z']:
-                    instruction = single_pauli(pstr)
-                    instruction["qubits"] = [qubit]
-                    circuit.append(instruction)
-                elif pstr != 'I':
-                    raise NoiseError("Invalid Pauli string.")
-            if circuit == []:
-                prob_identity += prob
-            else:
-                pauli_circuits.append(circuit)
-                pauli_probs.append(prob)
-    if prob_identity > 0:
-        pauli_circuits.append([{"name": "id", "qubits": [0]}])
-        pauli_probs.append(prob_identity)
+    # TODO: remove these two lines after noise model compiler (unroller) is introduced in the future
+    if num_qubits > 1:
+        paulis = [pauli.to_instruction().definition for pauli in paulis]
 
-    error = QuantumError(
-        zip(pauli_circuits, pauli_probs), number_of_qubits=num_qubits)
-    return error
+    if not standard_gates:
+        warnings.warn(
+            '"standard_gates" option has been deprecated as of qiskit-aer 0.10.0'
+            ' and will be removed no earlier than 3 months from that release date.',
+            DeprecationWarning, stacklevel=2)
+        paulis = [Operator(pauli).to_instruction() for pauli in paulis]
```

Comment:
Might as well just raise the warning that this option does nothing anymore and not change behavior